src/data/security-quiz.ts

```diff
@@ -286,6 +286,16 @@ const securityQuiz = [
       "RSA, named after its creators (Rivest, Shamir, and Adleman), is an encryption technique that takes advantage of the fact that factoring large prime numbers is hard.",
     Link: "https://www.freecodecamp.org/news/understanding-encryption-algorithms/",
   },
+  {
+    Question: "Which of these is NOT a OWASP top 10 vulnerability?",
+    Answer: "Buffer Vulnerability",
+    Distractor1: "Sensitive Data Exposure", 
+    Distractor2: "Broken Authentication",
+    Distractor3: "Injection Vulnerability",
+    Explanation: 
+      "The OWASP Top 10 guidelines is a standard awareness document that explains the top 10 most common vulnerabilities. A buffer vulnerability is a important security risk, but not part of OWASP's Top 10.",
```

Comment:
Let's shorten this a little bit

```suggestion
      "A buffer vulnerability is an important security risk, but it was not part of OWASP's 2021 Top 10 common vulnerabilities.",
```